src/nxdoc/nuxeo-add-ons/nuxeo-coldstorage/nuxeo-coldstorage-installation.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Installation and Configuration
+description: Learn how to install and configure the Nuxeo Coldstorage addon.
+review:
+    comment: ''
+    date: ''
+    status: ''
+labels:
+    - storage
+    - glacier
+    - coldstorage
+toc: true
+tree_item_index: 300
+---
+
+## Pre-requisites
+- The [Amazon S3 Online Storage]({{page version='' space='nxdoc' page='amazon-s3-online-storage'}}) addon needs to be configured on your instance
+
+## Amazon S3
+Nuxeo Platform with Nuxeo Coldstorage addon requires the usage of 2 Amazon buckets:
+
+A standard S3 bucket as for any other standard Nuxeo instance: this bucket is used to store the standard documents. You can refer to [Amazon S3 Online Storage]({{page version='' space='nxdoc' page='amazon-s3-online-storage'}}) documentation to configure this bucket.
+
+A secondary standard S3 bucket: this bucket is used to store the documents with glacier class. Compared to the standard bucket, a lifecycle must be defined to handle the document class change. You can refer to the [AWS documentation here](https://docs.aws.amazon.com/AmazonS3/latest/userguide/lifecycle-transition-general-considerations.html) for reference.
+
+## Configuration
+Make sure that all configuration values are set in the `nuxeo.conf`:
+
+```
+nuxeo.aws.accessKeyId=
+nuxeo.aws.secretKey=
+nuxeo.aws.region=
+nuxeo.s3storage.bucket_prefix=
+nuxeo.s3storage.bucket=
+nuxeo.aws.glacier.bucket=
+nuxeo.aws.glacier.bucket_prefix=
+```
+
+There are some other optional configurations:
+
+ Dor the scheduler jobs that check on async operation status - by default, the time is every 7 minutes but it can be customized accordingly to the needs of the client.
```

Comment:
```suggestion
 For the scheduler job that check on async operation status - by default, the time is every 7 minutes but it can be customized accordingly to the needs of the client.
```

Reply:
A reader that doesn't know much about the implementation might not understand what we mean by "check on async operation status"? What operation? A bit more detail would help.